Description: Add file which is missing from the 3.18.2 dist tarball
Bug: https://bugzilla.gnome.org/show_bug.cgi?id=757952
Index: gnome-online-accounts-3.18.2/data/org.gnome.Identity.service.in
===================================================================
--- /dev/null	1970-01-01 00:00:00.000000000 +0000
+++ gnome-online-accounts-3.18.2/data/org.gnome.Identity.service.in	2015-11-11 16:07:21.825339094 +0100
@@ -0,0 +1,3 @@
+[D-BUS Service]
+Name=org.gnome.Identity
+Exec=@libexecdir@/goa-identity-service
